main.go

```diff
@@ -81,13 +90,14 @@ func parseBuckets(buckets string) ([]float64, error) {
 
 func main() {
 	var (
+		configFile    = kingpin.Flag("config.file", "Optional smokeping_prober configuration file.").Default("smokeping_prober.yml").String()
```

Comment:
I am not sure I would put a default so that users explicitly know when they use the file or not and are not surprised.

Reply:
I wasn't sure which way I wanted to go with that.